modules/enum__secrets/main.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+import argparse
+from botocore.exceptions import ClientError
+import os
+
+# When writing a module, feel free to remove any comments, placeholders, or
+# anything else that doesn't relate to your module.
+
+module_info = {
+    # Name of the module (should be the same as the filename).
+    'name': 'enum__secrets',
+
+    # Name and any other notes about the author.
+    'author': 'Nick Spagnola From RSL',
+
+    # Category of the module. Make sure the name matches an existing category.
+    'category': 'ENUM',
+
+    # One liner description of the module functionality. This shows up when a
+    # user searches for modules.
+    'one_liner': 'enumerates and dumps secrets from AWS Secrets Manager and AWS parameter store',
+
+    # Full description about what the module does and how it works.
+    'description': 'This module will enumerate secrets in AWS Secrets Manager and AWS Systems manager parameter store.',
+
+    # A list of AWS services that the module utilizes during its execution.
+    'services': [],
+
+    # For prerequisite modules, try and see if any existing modules return the
+    # data that is required for your module before writing that code yourself;
+    # that way, session data can stay separated and modular.
+    'prerequisite_modules': [],
+
+    # External resources that the module depends on. Valid options are either
+    # a GitHub URL (must end in .git), or a single file URL.
+    'external_dependencies': [],
+
+    # Module arguments to autocomplete when the user hits tab.
+    'arguments_to_autocomplete': ['--regions'],
```

Comment:
Maybe add arguments to optionally choose to do one or the other (SSM/Secret Manager) instead of both every time